udata/core/dataset/api.py

```diff
@@ -288,12 +288,13 @@ def post(self, dataset):
 @api.doc(parser=upload_parser, **common_doc)
 class UploadCommunityResources(UploadMixin, API):
```

Comment:
I've just checked, these are different APIs and names differs with the trailing `s`:
- one allows to create by upload (this is why owner is set)
- the other allows to update by upload (owner is already set but we check permission)

I propose we let like this on master as it will be a lot more than the initial fix and we rename it in refactoring on dev because:
- Swagger operation IDs, which are used by the admin, need to change in consequence (and so the admin code
- we need to do a big cleanup/coherence check on these APIs (which will probably breaks backward compatibility and so need a new API version)